### PR TITLE
Renamed toStringFormat -> toString in DateTime and TimeSpan

### PR DESCRIPTION
**Pull request by sebastianbaginski**
@simonmika told me that we can make a PRs for the issues in "PR Debt" and "Cleanup" milestone in Kean.
Fixes https://github.com/cogneco/ooc-kean/issues/773

### CODE REVIEW

File: source/base/TimeSpan.ooc [hunk 1]
```diff
@@ -54,14 +54,14 @@ TimeSpan: cover {
 	//  %h - hours (<24)
 	//  %m - minutes (<60)
 	//  %s - seconds (<60)
-	//  %z - milliseconds (<1000)	
+	//  %z - milliseconds (<1000)
```
Comment (marcusnaslund) [inline source/base/TimeSpan.ooc]: This is a copy of the line below for some reason, so just remove the whole thing.

Comment (sebastianbaginski) [inline source/base/TimeSpan.ooc]: Updated.

File: source/base/DateTime.ooc [hunk 2]
```diff
@@ -112,7 +112,7 @@ DateTime: cover {
 	//	%ss - second
 	//	%zzzz - millisecond
 	// <param name="format">output format specification</param>
-	toStringFormat: func (format: String) -> String {
+	toString: func ~format (format: String) -> String {
```
Comment (davidhesselbom) [inline source/base/DateTime.ooc]: Hmmm... but why not
```ooc
toString: func (format := This defaultFormat) -> String {
```
?

Comment (marcusnaslund) [inline source/base/DateTime.ooc]: Why? `toString()` would already do this.

Comment (davidhesselbom) [inline source/base/DateTime.ooc]: Because then you don't need 2 overloads, all you need is the one with the default parameter.

Comment (marcusnaslund) [inline source/base/DateTime.ooc]: Oh, right, then we don't need `toString`.

Comment (davidhesselbom) [inline source/base/DateTime.ooc]: Or rather, we don't need `toString ~format`.

Comment (sebastianbaginski) [inline source/base/DateTime.ooc]: True. Updated.

**Pull request merged**